src/main/scala/cromwell/Main.scala

```diff
@@ -43,6 +45,22 @@ object Main extends App {
    * array becoming null in "new Main(args)" when used with: sbt 'run run ...'
    */
   new Main().runAction(args)
+
+  /**
+    * If a cromwell server is going to be run, makes adjustments to the default logback configuration.
+    * Overwrites LOG_MODE system property used in our logback.xml, _before_ the logback classes load.
+    * Restored from similar functionality in
+    *   https://github.com/broadinstitute/cromwell/commit/2e3f45b#diff-facc2160a82442932c41026c9a1e4b2bL28
+    * TODO: Logback is configurable programmatically. We don't have to overwrite system properties like this.
```

Comment:
Someday, yes. Setting system properties is technically possible, but feels strange. Also, I'm not sure the exact details at the moment, but cromwell seems to exists as both a library and an executable. There should be some default logging at various levels for the cromwell library, but maybe only default logging appenders would be setup within the executable.